lib/rouge/lexers/syzlang.rb

```diff
@@ -27,7 +27,7 @@ def self.keywords_type
 
       comment = /#.*$/
       inline_spaces = /[ \t]+/
-      eol_spaces = /[\n\r]+/
+      _eol_spaces = /[\n\r]+/
```

Comment:
For cases like this, I'd like to investigate why these aren't used and either remove them or use them, rather than adding an underscore like this. I... could be convinced otherwise though.

Reply:
I don't care either way. I can remove instead.